yarn.lock

```diff
@@ -27,16 +27,16 @@
     "@babel/highlight" "^7.10.4"
 
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4", "@babel/code-frame@^7.12.13", "@babel/code-frame@^7.5.5":
-  version "7.12.13"
```

Comment:
Doesn't feel like this file should be changing, or changing that much.  Thinking something happened since the x-pack/yarn.lock file is also in an iffy state.

Reply:
Yes, this was a mistake :(